src/services/LocalStorageKeystore.ts

```diff
@@ -65,6 +65,7 @@ export interface LocalStorageKeystore {
 	): Promise<[CryptoKey, WrappedKeyInfo]>,
 	upgradePrfKey(prfKeyInfo: WebauthnPrfEncryptionKeyInfo, promptForPrfRetry: () => Promise<boolean | AbortSignal>): Promise<[EncryptedContainer, CommitCallback]>,
 	getCachedUsers(): CachedUser[],
+	getUserHandleB64u(): string,
```

Comment:
This might be null, so it should be declared as nullable (both here and in the implementation).

```suggestion
	getUserHandleB64u(): string | null,
```